pygotham/frontend/templates/talks/detail.html

```diff
@@ -15,7 +15,17 @@
         {{ talk.user }}
       </a>
     </h4>
-
+    {% for presenter in talk.co_presenters.all() %}
+      <h4>
+      {% if presenter.user %}
```

Comment:
If we are contacted for a co-presenter, but the user does not make an account and we only have a name/email.